srcpkgs/scrcpy/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'scrcpy'
 pkgname=scrcpy
-version=1.3
+version=1.5
 revision=1
 build_style=meson
 configure_args="-Db_lto=true
```

Comment:
please remove the -Db_lto=true entry from configure_args= since that is default on the meson build_style